codeship-steps.yml

```diff
@@ -1,9 +1,24 @@
-- name: Deployment Test
+- name: Deployment Test via Ansible Container
+  service: heroku_on_ansible
+  command: codeship_heroku_deploy -f /app -N codeship-test-ruby-app
+- name: Deployment Test via Ansible Container with extra command
+  service: heroku_on_ansible
+  command: codeship_heroku_deploy -f /app -N codeship-test-ruby-app -d 'echo "foo"'
+- name: Deployment Test via Ansible Container with URL check
+  service: heroku_on_ansible
+  command: codeship_heroku_deploy -f /app -N codeship-test-ruby-app -c
+- name: Make sure that files are executable in "legacy paths"
+  service: heroku_on_ansible
+  command: bash -c 'test -x "$(which check_access_to_heroku_app)" && test -x "$(which heroku_run)"'
+- name: Deployment Test via plain container
   service: heroku
-  command: codeship_heroku deploy /app/test codeship-deployment-test
+  command: codeship_heroku_deploy -f /app -N codeship-test-ruby-app
+- name: Deployment Test via plain container with old entry point/command
+  service: heroku
+  command: codeship_heroku deploy /app codeship-test-ruby-app
 - name: Heroku Run Test
   service: heroku
-  command: heroku run --exit-code --app codeship-deployment-test -- true
+  command: heroku run --exit-code --app codeship-test-ruby-app -- true
```

Comment:
changing up the used app to a sandbox app, which fixes https://github.com/codeship-library/heroku-deployment/issues/18